src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-box search-box--big">
+      <input
+        class="search-txt search-txt--big"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-box search-box--small">
+      <input
+        class="search-txt search-txt--small"
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Keep your attributes correctly formatted. The `input` element on lines 30-35 should have each attribute on a new line with the correct indentation.